libraries/botbuilder-ai-orchestrator/src/orchestratorRecognizer.ts

```diff
@@ -216,7 +216,13 @@ export class OrchestratorRecognizer extends AdaptiveRecognizer implements Orches
 
             // if top scoring intent is less than threshold, return None
             if (topScore < this.unknownIntentFilterScore) {
-                recognizerResult.intents.None = { score: 1.0 };
+                recognizerResult.intents['None'] = { score: 1.0 };
+                // add all scores
+                results.reduce(function (intents, result) {
```

Comment:
This doesn't appear to be adding anything, am I missing something? This could be `.forEach(...)`.